plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -113,6 +115,11 @@ func (r *NodeAuthorizer) Authorize(attrs authorizer.Attributes) (authorizer.Deci
 				return r.authorizeCreateToken(nodeName, serviceAccountVertexType, attrs)
 			}
 			return authorizer.DecisionNoOpinion, fmt.Sprintf("disabled by feature gate %s", features.TokenRequest), nil
+		case leaseResource:
+			if r.features.Enabled(features.NodeLease) {
+				return r.authorizeLease(nodeName, attrs)
+			}
+			return authorizer.DecisionNoOpinion, fmt.Sprintf("disabled by feature gate %s", features.NodeLease), nil
 		}
```

Comment:
oh, also need a rule in `NodeRules()` for those still using RBAC only with nodes:

```go
// Node leases
if utilfeature.DefaultFeatureGate.Enabled(features.CSIPersistentVolume) {
	nodePolicyRules = append(nodePolicyRules, rbacv1helpers.NewRule("get","list","watch","create","update","patch","delete").Groups("coordination.k8s.io").Resources("leases").RuleOrDie())
}
```

Reply:
Do we need list and watch permission in that list? They aren't allowed by the authz bits.

Reply:
> Do we need list and watch permission in that list? They aren't allowed by the authz bits.
if they're not needed, you can omit them. for some reason I thought the leader election bits watched the resource

Reply:
`NodeRules()` is just for `system:node` right? Which just applies to Kubelet right? Kubelet doesn't need to watch its own lease.

Reply:
> NodeRules() is just for system:node right? Which just applies to Kubelet right? Kubelet doesn't need to watch its own lease.

yes